src/pages/mixins/BrowsePageMixin.js

```diff
@@ -11,6 +13,22 @@ const BrowsePage = {
       return this.$store.state.isFilterVisible;
     },
   },
+  beforeRouteLeave(to, from, next) {
```

Comment:
There is already code that does this: https://github.com/creativecommons/cccatalog-frontend/blob/develop/src/pages/HomePage.vue#L23

Probably when we rewrote the filters' code, this piece of code broke, so I think it's best if you just commit the `CLEAR_FILTERS` there